test/bin/scenario_container.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
```

Comment:
That's a great idea. Can you add it to the other two files we source in `tests/bin`?